### PR TITLE
Support select_cc with fat pointer type

### PR DESCRIPTION
**Pull request by lialan**

### CODE REVIEW

**Comment by zksync-admin-bot2** [comment]
Benchmark results:
```
╔═╡ Size (-%) ╞═════════════════════╡ All ╞═╗
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╠═╡ Cycles (-%) ╞═══════════════════╡ All ╞═╣
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╚═══════════════════════════════════════════╝

╔═╡ Size (-%) ╞═══════════════╡ Real life ╞═╗
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╠═╡ Cycles (-%) ╞═════════════╡ Real life ╞═╣
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╚═══════════════════════════════════════════╝

```

File: llvm/test/CodeGen/SyncVM/selectcc_fatptr.ll [hunk 1]
```diff
@@ -0,0 +1,18 @@
+; RUN: llc < %s | FileCheck %s
+
+target triple = "syncvm"
+target datalayout = "E-p:256:256-i256:256:256-S32"
+
+
```
Comment (akiramenai) [inline llvm/test/CodeGen/SyncVM/selectcc_fatptr.ll]: please remove redundant empty lines

**Review by akiramenai — APPROVED** [review]

File: llvm/test/CodeGen/SyncVM/selectcc_fatptr.ll (hunk 1 above)
Comment (lialan) [reply llvm/test/CodeGen/SyncVM/selectcc_fatptr.ll]: removed.

**Comment by zksync-admin-bot2** [comment]
Benchmark results:
```
╔═╡ Size (-%) ╞═════════════════════╡ All ╞═╗
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╠═╡ Cycles (-%) ╞═══════════════════╡ All ╞═╣
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╚═══════════════════════════════════════════╝

╔═╡ Size (-%) ╞═══════════════╡ Real life ╞═╗
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╠═╡ Cycles (-%) ╞═════════════╡ Real life ╞═╣
║ Mean                                0.000 ║
║ Best                                0.000 ║
║ Worst                               0.000 ║
║ Total                               0.000 ║
╚═══════════════════════════════════════════╝

```

**Pull request merged**